client/src/components/new-request/analyze-results.vue

```diff
@@ -519,6 +519,11 @@ export default class AnalyzeResults extends Vue {
       showNextLines: true
     }
   }
+  get headerText () {
+    let issues = this.json.issues
+    return issues.length === 1 && issues[0].issue_type === 'excess_words' ? 'Further Review Required'
```

Comment:
Is there any chance of `issues` being null or undefined -- should you use conditional chaining on the length check?

Reply:
I have seen it as an empty array if there are no issues. Line 496 would have given issue at some point otherwise I think